app_list.js

```diff
@@ -34,6 +42,9 @@ app.controller("app_list", function ($scope, $http) {
 
 
     $scope.categoryFilter = function (event) {
+        if ($scope.category !== null) {
```

Comment:
Can't this condition be removed and let `$scope.category` be made null directly?